src/controller.rs

```diff
@@ -96,7 +96,7 @@ impl Controller for FoxBox {
 
     fn run(&mut self, shutdown_flag: &AtomicBool) {
         debug!("Starting controller");
-
+        info!("sqlite location: {}", self.profile_service.path_for("users_db.sqlite"));
```

Comment:
That's a hardcoded string. I'd either remove this line or extract the path retrieval in a private function.